### PR TITLE
updated some more gems

### PR DESCRIPTION
**Pull request by wasi**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -8,7 +8,7 @@ gem 'bundler', '>= 1.7.0'
 gem 'rmagick'
 gem 'carrierwave-aws'
 gem 'devise'
-gem 'friendly_id', '~> 5.0.0'
+gem 'friendly_id', '~> 5.1.0'
```
Comment (beanieboi) [inline Gemfile]: maybe just '~> 5.1'

File: Gemfile [hunk 2]
```diff
@@ -16,13 +16,13 @@ gem 'omniauth'
...
 gem 'react-rails', '~> 1.0.0.pre', github: 'reactjs/react-rails'
-gem 'sass-rails', '~> 4.0.3'
+gem 'sass-rails', '~> 5.0.1'
 gem 'sdoc', '~> 0.4.0',          group: :doc
 gem 'uglifier', '>= 1.3.0'
 gem 'acts-as-taggable-on', '~> 3.4'
 gem 'cancancan', '~> 1.10'
 gem 'newrelic_rpm'
-gem 'rollbar', '~> 1.2.7'
+gem 'rollbar', '~> 1.4.4'
```
Comment (beanieboi) [inline Gemfile]: i would drop the constraint completly

File: Gemfile [hunk 3]
```diff
@@ -42,7 +42,7 @@ end
 
 group :development, :test do
   gem 'faker'
-  gem 'rspec-rails', '~> 3.1.0'
+  gem 'rspec-rails', '~> 3.2.0'
```
Comment (beanieboi) [inline Gemfile]: this should be fine as well '~> 3.0'

**Comment by wasi** [comment]
@beanieboi good points. I changed it in another PR

**Pull request merged**